templates/SilverStripe/Admin/LeftAndMain.ss

```diff
@@ -1,20 +1,29 @@
 <!DOCTYPE html>
 <html lang="$Locale.RFC1766">
-	<head>
-	<% base_tag %>
-	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
-	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-	<title>$Title</title>
+    <head>
+    <% base_tag %>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>$Title</title>
 </head>
 <body class="loading cms" data-frameworkpath="$ModulePath(silverstripe/framework)"
-	data-member-tempid="$CurrentMember.TempIDHash.ATT"
+    data-member-tempid="$CurrentMember.TempIDHash.ATT"
 >
-	<% include SilverStripe\\Admin\\CMSLoadingScreen %>
+    <% include SilverStripe\\Admin\\CMSLoadingScreen %>
 
-	<div class="cms-container" data-layout-type="custom">
-		$Menu
-		$Content
-		$PreviewPanel
+    <a class="cms-container-skip-link"
+       href="#cms-container-skip-link-target"
+       tabindex="0"
+    ><%t SilverStripe\\Admin\\LeftAndMain.SkipLink "Skip to main content" %></a>
+
+    <div class="cms-container" data-layout-type="custom">
+        $Menu
+        <div id="cms-container-skip-link-target"
+             class="cms-container-skip-link-target"
+             tabindex="-1"
```

Comment:
Adding tabindex="-1" to the skip link's target element makes it programmatically focusable, without being tabbable

Reply:
Why `role="main"` and not use a `<main>` tag?